extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -202,7 +202,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         'iniLoad': {
           'on': 'visible',
           'request': 'fvt',
-          'selector': 'amp-ad',
+          'selector': 'body',
           'selectionMethod': 'closest',
```

Comment:
Does `closest` matter / make sense for `body`?

Reply:
Hmm...no. Looking at the implementation of that, it seems that the options are "closest", "scope", and "null". Seems like "null" does what we want, and that simply omitting "selectionMethod" altogether achieves that.